Microsoft Defender/microsoft-defender-for-endpoints/_meta/smart-descriptions.json

```
@@ -168,5 +200,71 @@
         "field": "host.name"
       }
     ]
+  },
+  {
+    "value": "{service.name} has associated the file {file.name} with the threat: {microsoft.defender.threat.family} (sha1: {file.hash.sha1}).",
+    "conditions": [
+      {
+        "field":"microsoft.defender.entity.type",
+        "value": "File"
+      },
+      {
+        "field":"microsoft.defender.threat.family"
+      },
+      {
+        "field":"file.hash.sha1"
+      },
+      {
+        "field":"file.name"
+      },
+      {
+        "field":"service.name"
+      }
+    ]
+  },
+  {
+    "value": "An alert ({microsoft.defender.alert.id}) has been raised for application {action.properties.Application} ",
+    "conditions": [
+      {
+        "field":"microsoft.defender.entity.type",
+        "value": "CloudApplication"
+      },
+      {
+        "field":"microsoft.defender.alert.id"
+      },
+      {
+        "field":"action.properties.Application"
+      }
+    ]
+  },
+  {
+    "value": "User: {user.name} has raised alert {microsoft.defender.alert.id}.",
```

```suggestion
    "value": "Alert {microsoft.defender.alert.id}  was raised for user {user.name}",
```

I suggest that we invert the sentence